common/interfaces/card_interface/nfc.h

```diff
@@ -181,7 +182,7 @@ ISO7816 nfc_unpair();
  *
  * @note
  */
-ISO7816 nfc_list_all_wallet(uint8_t recv_apdu[], uint16_t *recv_len);
+ISO7816 nfc_list_all_wallet(wallet_list_t *wallet_list);
```

Comment:
Please update the doxygen comment accordingly

Reply:
Updated the comment